privacyidea/api/validate.py

```diff
@@ -616,12 +617,22 @@ def poll_transaction(transaction_id=None):
                            if challenge.is_valid()]
     answered_challenges = extract_answered_challenges(matching_challenges)
 
+    decline_challenges = []
     if answered_challenges:
         result = True
         log_challenges = answered_challenges
+        details = {"challenge_declined": "accept"}
```

Comment:
I think this has to be

~~~~
details = {"challenge_status": "accept"}
~~~~

challenge_declined -> challenge_status